### ISSUE TITLE
unit testing for accuracy of d-scores

### ISSUE THREAD

**Issue by wjakethompson** [OttaviaE/implicitMeasures#5]
From the README, it appears that one of the main motivations for this package is that there are often computation errors in the calculation of the D-scores, which raise replicability concerns. I agree that this is a great reason to write a package!

However, when looking at the unit tests for the `computeD()` function, it doesn't look like there are any tests for the accuracy of the calculated values. Right now, the tests check for errors, check the class of return values, and the size of the output (i.e., expected number of rows). These are all important tests, but I think it would be beneficial to also include tests for the accuracy of values. That is, the D-scores returned by the function are what we expect them to be.

This is important to:

1. Ensure the validity of the package
2. Protect against future changes

Presumably, this package will need to be updated at some point in the future as dependencies also get updated. At that time, it would be helpful to have these unit tests in place to ensure that future changes don't mess with any of the calculations, which would then threaten the validity of scores generated from this package.

**Comment by wjakethompson** [comment]
this is for openjournals/joss-reviews#2394

**Comment by OttaviaE** [comment]
I totally agree with what you pointed out. I stored the results from the computation of the IAT and SC-IAT D-score in .rda objects in the `data` folder, and then I used them to test the functions for the computation of the IAT/SC-IAT D-score, and, so far they result in the same exact scores. The new test functions are `test-computeD-replicability` and `test-SCIAT-replicate`.
Thank you for pointing that out, it was really helpful!
